src/ebay-pagination/pagination.tsx

```diff
@@ -105,7 +105,7 @@ const EbayPagination: FC<PaginationProps> = ({
             const newProps = {
                 current, disabled, href,
                 type: isSeparator ? 'separator' : type,
-                children: isDot ? <EbayIcon name="overflow" height="18px" focusable={false} /> : text,
+                children: isDot ? <EbayIcon name="overflowHorizontal24" height="18px" focusable={false} /> : text,
```

Comment:
Would `overflowHorizontal16` or `overflowHorizontal20` work here? Hopefully we can avoid overriding sizes.

Reply:
you're right, missed those override attempts, we don't need them.